client/my-sites/exporter/exporter.jsx

```diff
@@ -43,6 +43,7 @@ export default React.createClass( {
 		const siteId = this.props.site.ID;
```

Comment:
Good point. I think my original idea was to separate actions (function props) from state, but it's not all that useful. Will fix that up